src/js/apps/forms/form/form_app.js

```diff
@@ -98,22 +93,30 @@ export default App.extend({
   onClickPrintButton() {
     Radio.request(`form${ this.form.id }`, 'print:form');
   },
-  onChangeSidebar(state, sidebar) {
-    this.stopChildApp('patient');
-    Radio.request('sidebar', 'close');
+  onChangeState(model) {
+    if (model.hasChanged('isExpanded') || model.hasChanged('isActionSidebar')) {
```

Comment:
rather than wrap the whole thing, let's make this a guard:
`if (!hasChanged('isExpanded.. &&!hasChange('isAction..) return;`